builtin/credential/ldap/backend.go

```diff
@@ -281,18 +291,20 @@ func (b *backend) getLdapGroups(cfg *ConfigEntry, c *ldap.Conn, userDN string, u
 	ldapMap := make(map[string]bool)
 
 	if cfg.GroupFilter == "" {
-		b.Logger().Printf("[WARN] auth/ldap: GroupFilter is empty, will not query server")
+		b.Logger().Warn("auth/ldap: GroupFilter is empty, will not query server")
```

Comment:
IsWarn check is missing here and in the next log message as well.

As a general comment, it is a real pain to check IsDebug or IsWarn before printing anything.

Is it not possible that the Logger().Debug() will only print if Debug is enabled. So we can conveniently skip the check.